packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-linking-step/wallet-linking-step.scss

```
@@ -38,8 +37,14 @@
         margin-top: 5.6rem;
 
         &-text {
+            text-align: center;
             padding: 0.4rem 0.8rem;
             border-radius: 24px;
```

```suggestion
            border-radius: $BORDER_RADIUS * 6;
```
FYI: $BORDER_RADIUS: 4px

---

Hi @aizad-deriv, please correct me if I am wrong, `24px` is not a common constant used in our codebase. Also, it is just inconvenient for the reader to comprehend when they first, need to find the value of `$BORDER_RADIUS` and then multiply it by 6 to know the actual value.
I have tried to write code in this PR to make it less confusing for the next person to work on it.
The [same](https://github.com/binary-com/deriv-app/pull/10145#discussion_r1335661124.) applies here.

I hope you get my point 🙏.

---

You can find `$BORDER_RADIUS: 4px` in `constants.scss`. But if you say you want todo it for readability then it's fine.